src/controllers/edit-service-name/get.controller.test.js

```diff
@@ -17,7 +18,7 @@ describe('Controller: editServiceName, Method: get', () => {
       mockResponses.response = sinon.spy()
       res = {}
       req = {
-        service: new Service({ external_id: random.randomUuid(), name: 'Example Service', service_name: { en: 'Example En Service', cy: 'Example Cy Service' } })
+        service: new Service(validServiceResponse({ external_id: random.randomUuid(), name: 'Example Service', service_name: { en: 'Example En Service', cy: 'Example Cy Service' } }))
```

Comment:
What's the advantage of using `validServiceResponse` here as opposed to what we have on the left?

Reply:
Previously, the Service class would just set any missing fields to undefined when instantiated. Now it will throw an error if you try to create a service without mandatory fields so the tests that 'create' a service needed updating to use a fixture